coordinator/cql/src/main/java/org/apache/cassandra/stargate/transport/internal/messages/OptionsMessage.java

```diff
@@ -64,13 +63,6 @@ protected CompletableFuture<? extends Response> execute(long queryStartNanoTime)
     supported.put(StartupMessage.COMPRESSION, compressions);
     supported.put(StartupMessage.PROTOCOL_VERSIONS, ProtocolVersion.supportedVersions());
 
-    if (connection.getVersion().supportsChecksums()) {
-      ChecksumType[] types = ChecksumType.values();
-      List<String> checksumImpls = new ArrayList<>(types.length);
-      for (ChecksumType type : types) checksumImpls.add(type.toString());
-      supported.put(StartupMessage.CHECKSUM, checksumImpls);
-    }
-
```

Comment:
removed? is this breaking change for the old protocols?

Reply:
They are no logger supported in 4.0.10 OptionMessage.java for protocol v5.